provisioning/miniprov/samples/exampleV1.json

```diff
@@ -32,7 +32,7 @@
   "corosync-pacemaker": {
     "cluster_name": "cortx_cluster",
     "user": "hacluster",
-    "secret": "gAAAAABgDsD7pV2BxPXu6NeIIrVCwptUHbZaODtAAgo9Ik778Qz2lgu3xZzJUk7qNlU5ZMHZYq5HKoD500Df359jJ4JZ9z1KcQ=="
+    "secret": "STUB"
```

Comment:
On second thought, Hare does not read these keys, why not just remove them? we can have just skeletons for all these keys having secrets in them.

Reply:
Because it is not us who created this example JSON. The less we need to tweak it, the closer it to the "original".